### ISSUE TITLE
windows: ctrl+Enter commit overridden by this extension

### ISSUE THREAD

**Issue by VDFaller** [innoverio/vscode-dbt-power-user#135]
## issue
Currently when having this installed, if I put in a commit message and press control+enter, `dbtPowerUser.executeSQL` runs.  

## Possible fix
In [package.json](https://github.com/innoverio/vscode-dbt-power-user/blob/master/package.json#L204)

change

```json
        "when": "resourceLangId == jinja-sql"
```

to 

```json
        "when": "editorTextFocus && resourceLangId == 'jinja-sql'"
```

**Comment by mdesmet** [comment]
Thanks for the help! Can you create a PR?

**Comment by VDFaller** [comment]
Yeah, I didn't see any PR requirements.  Anything I need to do other than this change?  Testing and what not?

**Comment by mdesmet** [comment]
Currently this extension doesn't have any tests. 

If you are eager to work on testing the extension, you may contact me on dbt slack. I have some ideas but don't have the time to implement them.
